### PR TITLE
migration to Web-KYC, added sdkVersion and sdkType parameters to url

### PR DESCRIPTION
**Pull request by pawel-zak**
removed passbase dependencies and implementation

### CODE REVIEW

File: rampsdk/src/main/java/network/ramp/sdk/ui/activity/RampWidgetActivity.kt [hunk 1]
```diff
@@ -40,12 +43,15 @@ internal class RampWidgetActivity : AppCompatActivity(), Contract.View {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = WidgetActivityBinding.inflate(layoutInflater)
-
+        WebView.setWebContentsDebuggingEnabled(true);
```
[the remark this answers is not in the document]
Comment (pawel-zak) [reply rampsdk/src/main/java/network/ramp/sdk/ui/activity/RampWidgetActivity.kt:46]: Yes, it should. Thanks for point out

**Review by jakubsta — DISMISSED** [review]

File: rampsdk/build.gradle [hunk 2]
```diff
@@ -86,7 +85,7 @@ afterEvaluate {
                 from components.release
                 groupId = 'com.github.RampNetwork'
                 artifactId = 'ramp-sdk-android'
-                version = '2.0.0'
+                version = '2.1.0'
```
Comment (jakubsta) [inline rampsdk/build.gradle:88]:
```suggestion
                version = '3.0.0'
```

File: rampsdk/build.gradle [hunk 3]
```diff
@@ -22,8 +22,8 @@ android {
     defaultConfig {
         minSdkVersion 21
         targetSdkVersion 32
-        versionCode 15
-        versionName "2.0.0"
+        versionCode 16
+        versionName "2.1.0"
```
Comment (jakubsta) [inline rampsdk/build.gradle:26]:
```suggestion
        versionName "3.0.0"
```

File: rampsdk/src/main/java/network/ramp/sdk/ui/activity/RampPresenter.kt [hunk 4]
```diff
@@ -162,79 +146,12 @@ internal class RampPresenter(
...
-                        kycInitPayload?.verificationId ?: 0,
-                        identityAccessKey
-                    )
-                )
-            )
-        view.sendPostMessage(eventJson)
-    }
+                "&deepLinkScheme=$DEEP_LINK_SCHEME" +
+                "&sdkType=ANDROID" +
+                "&sdkVersion=${BuildConfig.VERSION}"
```
Comment (jakubsta) [inline rampsdk/src/main/java/network/ramp/sdk/ui/activity/RampPresenter.kt:151]: Shouldn't it be `BuildConfig.VERSION_NAME`?

**Review by jakubsta — APPROVED** [review]

**Pull request merged**